Apache-Beam-Jobs.md

```diff
@@ -227,6 +228,82 @@ error_pcoll = (
 
 `Runner`s provide the `run()` method used to visit every node (`PValue`) in the pipeline's DAG by executing the edges (`PTransform`s) to compute their values.  At Oppia, we use `DataflowRunner` to have our `Pipeline`s run on the [Google Cloud Dataflow service](https://cloud.google.com/dataflow).
 
+## Writing Apache Beam Jobs
+
+Subclass the `base_jobs.JobBase` class and override the `run()` method.
+
+The name of your job class is presented to release coordinators, so make sure it is clear and concise:
+![Screenshot from 2021-09-28 09-05-10](https://user-images.githubusercontent.com/5094060/135092574-0bff536e-1b58-4b38-9358-c26f9096c8a3.png)
+
+**Job names should follow the convention: `<Verb><Noun>Job`.**
+
+For example:
+
+```python
+class WeeklyDashboardStatsComputationJob(base_jobs.JobBase):
+    """BAD: Name does not begin with a verb."""
+
+    def run(self):
+        ...
+
+
+class ComputeStatsJob(base_jobs.JobBase):
+    """BAD: Unclear what kind of stats are being computed."""
+
+    def run(self):
+        ...
+
+
+class CountExplorationInteractionsJob(base_jobs.JobBase):
+    """GOOD: Name starts with a verb and "exploration interactions" is unambiguous."""
+
+    def run(self):
+        ...
+```
+
+**Module names should follow the convention: `<entity>_<operation>_jobs.py`.**
+
+For example:
+* `blog_validation_jobs.py`
+* `dashboard_stats_computation_jobs.py`
+* `exploration_indexing_jobs.py`
+* `exploration_stats_regeneration_jobs.py`
+* `model_validation_jobs.py`
+
+**New modules must be imported in the `jobs/registry.py` file.**
+
+---
+
+The `run()` method must return a `PCollection[JobRunResult]`.
```

Comment:
I think it would be clearer to say "PCollection of JobRunResults" since not everyone is going to be familiar with the `PCollection[JobRunResult]` syntax